iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/StringSimilarityChecker.java

```diff
@@ -0,0 +1,37 @@
+package eu.dnetlib.iis.wf.affmatching.match.voter;
+
+import java.util.Collection;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Class containing helper methods checking strings similarity.
+ * 
+ * @author madryk
+ */
+class StringSimilarityChecker {
+
+
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Returns true if provided string values contains at least one
```

Comment:
...if the provided string values contain... NO 's' in verb when the noun is plural.